CMakeLists.txt

```diff
@@ -115,6 +115,30 @@ else()
   set(USE_UPDATER NO)
 endif()
 
+# Enable the 3D mapper by default on the three main supported OS
+# unless environmental variable WITH_3DMAPPER is defined and set to "NO" (case
+# insensitive).
+# WITH_3DMAPPER is an environmental value/variable (so could be a number, a
+# string, something else or not even exist).
+set(3DMAPPER_TEST $ENV{WITH_3DMAPPER})
+if((CMAKE_SYSTEM_NAME STREQUAL "Linux") OR (CMAKE_SYSTEM_NAME STREQUAL "Windows") OR (CMAKE_SYSTEM_NAME STREQUAL "Darwin"))
```

Comment:
Please also include FreeBSD it does work on that OS ARAICR - not sure right now what the "Word" is to test for that....

Reply:
Me neither - and it's not a supported platform, kind of don't to go dig for it...

Reply:
Okay, I´ll reboot into that in the next couple of days and work out what we need...